OpenRA.Platforms.Default/Sdl2PlatformWindow.cs

```diff
@@ -345,7 +345,12 @@ public Sdl2PlatformWindow(Size requestEffectiveWindowSize, WindowMode windowMode
 				Context = ctx;
 			}
 			else
+			{
+				if (Platform.CurrentPlatform == PlatformType.OSX)
+					SDL.SDL_SetHint(SDL.SDL_HINT_MAC_OPENGL_ASYNC_DISPATCH, "1");
```

Comment:
There's already a macOS specific branch above (line 267) - any reason why this can't be included there?